CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+First of all, kudos to everyone, who wants to share their ideas, comments, complaints, flames and other input with the mighty Robot! The Robot appreciates any kind of contribution, which helps him to get rid of humans, er… to take the burden off of them.
+
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change. The Robot will severely punish those who don’t comply.
+
+Please note the G1ANT.Robot has a code of conduct, so please follow it in all your interactions with the project.
+
+## Opening Issues
+
+1. Use a standard GitHub issue tool to send a message to the Robot and make him aware of an issue.
+2. Remember to attach screenshots, logs or other files that would help the Robot pinpoint the issue and/or reproduce it.
+3. If the Robot admits you are right, which is possible, but highly unlikely, you will be granted a right to make a pull request. Or the Robot will make the necessary changes himself, since he is inerrant.
```

Comment:
```suggestion
3. If the Robot admits you are right, which is possible, you will be granted a right to make a pull request. Or the Robot will make the necessary changes himself, since he is inerrant.
```